### ISSUE TITLE
vi & highlighting

### ISSUE THREAD

**Issue by helenkirk** [opencadc/arcade#45]
In both the older demo.canfar.net and current alpha prototype versions of ARCADE, highlighting doesn't work properly within a vi session.  Specifically, highlighting text makes vi immediately switch to 'visual' mode, and it is not possible to highlight & copy text (e.g., using cntrl-shift-c) to the Clipboard application in the alpha prototype.

**Comment by drtobybrown** [comment]
Is this still an issue?

**Comment by brianmajor** [comment]
Yes, it's still an issue.  It's possible to override the actions of ctrl
commands in a terminal environment, but it will take a lot of
experimentation to ensure a new set of commands doesn't interfere with
other command line tools and editors.  The action of selecting text must be
equivalent to a ctrl command that triggers visual mode in vi.
